pkg/kubelet/kubelet.go

```diff
@@ -3021,3 +3023,12 @@ func (kl *Kubelet) PrepareDynamicResources(pod *v1.Pod) error {
 func (kl *Kubelet) UnprepareDynamicResources(pod *v1.Pod) error {
 	return kl.containerManager.UnprepareDynamicResources(pod)
 }
+
+func (kl *Kubelet) warnCgroupV1Usage() {
+	cgroupVersion := kl.containerManager.GetNodeConfig().CgroupVersion
+	if cgroupVersion == 1 {
+		kl.recorder.Eventf(kl.nodeRef, v1.EventTypeWarning, events.CgroupV1, cm.CgroupV1MaintenanceModeWarning)
+		klog.V(2).Info(cm.CgroupV1MaintenanceModeWarning)
```

Comment:
```suggestion
		klog.V(2).InfoS("Warning: cgroup v1", "message", cm.CgroupV1MaintenanceModeWarning)
```
something like this.

Reply:
Thanks @pacoxu . Updated.